packages/p2p-v2/src/components/Modals/BlockUnblockUserFilterModal/BlockUnblockUserFilterModal.scss

```
@@ -0,0 +1,46 @@
+.p2p-v2-block-unblock-user-filter-modal {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    right: auto;
+    bottom: auto;
+    margin-right: -50%;
+    transform: translate(-50%, -50%);
+    border-radius: 8px;
+    background: #fff;
+    width: 32.8rem;
+    box-shadow: 0px 32px 64px 0px rgba(14, 14, 14, 0.14);
+
+    &__text {
+        margin: 2.4rem 0;
+
+        @include mobile {
+            margin: 1.6rem 0;
+        }
+    }
+
+    .p2p-v2-radio-group {
+        &.p2p-v2-sort-radiogroup {
+            height: 100%;
+            display: flex;
+            flex-direction: column;
+            justify-content: center;
+            align-items: flex-start;
+            margin: unset;
+        }
+        .p2p-v2-radio-group__item {
+            &.p2p-v2-sort-radiogroup {
```

Just a question though, where is there no space here?
```suggestion
            & .p2p-v2-sort-radiogroup {
```

---

here the div with sort-radiogroup is not a child of the other on.. it has to select both the classes